### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by hharchani**
- Use custom pure build
- Remove Kites' images
- Add synonyms for nick and organization

### CODE REVIEW

File: src/app/views/auth/profile.php [hunk 1]
```diff
@@ -53,7 +53,7 @@
         </div>
 
         <div class="pure-control-group">
-            <label for="organization"><?= __('Organization:') ?></label>
+            <label for="organization"><?= __('Organization / College / Institution / Company:') ?></label>
```
Comment (nisargjhaveri) [inline src/app/views/auth/profile.php]: Thoda lamba nahi ho ja raha?

Comment (nisargjhaveri) [inline src/app/views/auth/profile.php]: Updates?

Comment (hharchani) [inline src/app/views/auth/profile.php]: Kya karu iski jagah?
Sirf organization is not enough.

Comment (nisargjhaveri) [inline src/app/views/auth/profile.php]: Remove 'Institution', 'Organization / College / Company' is enough, I think.

**Comment by hharchani** [comment]
@nisargjhaveri this is updated BTW

**Comment by hharchani** [comment]
Merging this too.

**Pull request merged**